compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -2931,14 +2931,14 @@ OMR::CodeGenerator::setEstimatedLocationsForSnippetLabels(int32_t estimatedSnipp
    }
 
 uint8_t *
-OMR::CodeGenerator::emitSnippets(bool isWarm)
+OMR::CodeGenerator::emitSnippets()
    {
    uint8_t *codeOffset;
    uint8_t *retVal;
 
    for (auto iterator = _snippetList.begin(); iterator != _snippetList.end(); ++iterator)
       {
-      if ((*iterator)->isWarmSnippet() == isWarm)
+      if ((*iterator)->isWarmSnippet() == 0)
```

Comment:
Appreciate the staged approach since it makes each commit logically easier for the the reviewer to reason about.